src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,31 @@
+/* Experimental text primitive.
+ * Issues: color not changing, removeAttribute() not working, mixing primitive with regular entities fails
+ * Color issue relates to: https://github.com/donmccurdy/aframe-extras/blob/master/src/primitives/a-ocean.js#L44
```

Comment:
I wrote those comments for my variant of bmfont-text with A-Frame v0.3.0, so I can't say for sure if they're valid for Fernando's bmfont-text with A-Frame v0.4.0. My guess would be no.